src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -35,6 +36,8 @@
     private final GcsConfiguration gcsConfiguration;
     private final ConfigurationService configService;
 
+    private static final Long RETENTION_PERIOD = Duration.ofDays(30).getSeconds();
```

Comment:
unnecessary use of boxed type
```suggestion
    private static final long RETENTION_PERIOD = Duration.ofDays(30).getSeconds();
```

Reply:
I like the idea of having to manually opt-out to remove the policy -- it makes everyone aware of the danger of doing that. Also, good catch on the boxed type.